app/script/components/list/conversationListCallingCell.js

```diff
@@ -143,7 +143,15 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
     this.showParticipants = ko.observable(false);
 
     this.showVideoPreview = ko.pureComputed(() => {
-      return this.isVideoCall() && (this.multitasking.isMinimized() || !this.callIsConnected());
+      const nbOfOtherOngoingCalls = this.calls()
```

Comment:
We do not care about the number. You can just turn this into a boolean. Much easier on the naming as you can go with `hasOtherOngoingCall`

Reply:
Fair point :+1: